webots_ros2_epuck/worlds/epuck_world.wbt

```diff
@@ -1,9 +1,9 @@
-#VRML_SIM R2020a utf8
+#VRML_SIM R2020b utf8
```

Comment:
Be careful, you re-introduced the wrong header in this commit 😉

Reply:
Oh, no :facepalm: